src/geocoder/geocoder.ts

```diff
@@ -672,7 +669,9 @@ module powerbi.extensibility.geocoder {
                 return;
             }
 
-            GeocodeCallback = (data) => {
+            let guid =  "GeocodeCallback" + new Date().getTime();
```

Comment:
I'm not sure that date is a good choice to use as a GUID. I suppose there might be a conflict. Do you think it's reliable?